CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to the Openshift Builder
+
+The OpenShift Builder is the component that drives [OpenShift Builds](https://docs.okd.io/latest/dev_guide/builds/index.html),
+leveraging [Buildah](https://github.com/containers/buildah) and [Source-To-Image](https://github.com/openshift/source-to-image)
+to turn source code into images that can be deployed on Kubernetes.
+
+## Getting the code
+
+To get started, [fork](https://help.github.com/articles/fork-a-repo) the [openshift/builder](https://github.com/openshift/builder) repo.
+
+## Developing
+
+### Testing on an OpenShift Cluster
+
+The easiest way to test your changes is to launch an OpenShift 4.x cluster.
+First, go to [try.openshift.com](https://try.openshift.com) to obtain a pull secret and download the installer.
+Follow the instructions to launch a cluster on AWS.
+
+If you want the latest OpenShift installer and `oc` clients, go to the [openshift-release](https://openshift-release.svc.ci.openshift.org/) 
+page, select the channel and build you wish to install, and download the respective `oc` and `openshift-installer` binaries.
+There are three types of channels you can obtain the installer from:
+
+1. `4-stable` - these are stable releases of OpenShift 4, corresponding to GA or beta releases.
+2. `4.x.0-nightly` - nightly development releases, with payloads published to quay.io.
+3. `4.x.0-ci` - bleeding-edge releases published to the OpenShift CI imagestreams.
+
+**Note**: installs from the `4.x.0-ci` channel require a pull secret to `registry.svc.ci.openshift.org`, which is only available to Red Hat OpenShift developers.
```

Comment:
this is good doc but not something you're going to want to maintain in this repo as it inevitably changes over time.  i'd find a better home for it and point there.

Reply:
(to be clear i'm referring to all the steps about how to get a cluster installed)

Reply:
Agreed - I'll add a TODO to move this to origin.